core/src/test/java/org/infinispan/xsite/statetransfer/failures/SiteProviderTopologyChangeTest.java

```diff
@@ -238,7 +193,9 @@ private void doXSiteStateTransferDuringTopologyChange(TopologyEvent event) throw
       startStateTransfer(testCaches.coordinator, NYC);
       assertOnline(LON, NYC);
 
+      topologyManager.stopBlocking();
       blockedTopology.unblock();
+      topologyManager.unblockAll();
```

Comment:
IMO `unblockAll()` doesn't make sense without `stopBlocking()`, so it should either be controlled by a parameter or the default behaviour of `stopBlocking()`.

Currently `stopBlocking()` logs an error when there are unblocked topologies in the queue, which I think (now) is not a good compromise: it should either throw an exception or unblock all queued topologies (and log only a debug message).